diagnostic.md

```diff
@@ -8,6 +8,12 @@ Assume all code executes in strict mode.
 ## Question 1
 
 Please list the JavaScript primitives and give one example of each.
+string: "Spider-Man"
+number: "47"
```

Comment:
a Number type is just a number. The quotes you have around 47 would make it a string.